data/pathfinder/paizo/player_companion/bastards_of_golarion/bog_abilities.lst

```diff
@@ -55,21 +55,3 @@ Averaka											NAMEISPI:YES	CATEGORY:Background		TYPE:Region							VISIBLE:DI
 # ==============================
 
 The Canticle of Joy		KEY:Masterpiece ~ The Canticle of Joy				CATEGORY:Special Ability	TYPE:SpecialAttack.Supernatural.Masterpiece																																																													DESCISPI:YES	DESC:This agonizing eulogy gets into a listener's blood, and bestows on him the phantom sensation of all the agonies suffered by Zon-Kuthon's most pious worshipers. One target within range is cursed as though affected by bestow curse as long as the bard maintains the performance. Although this lullaby does have words, it is not a language-dependent effect. Use: 1 bardic performance per round. Action: 1 round.																																																			SOURCEPAGE:p.27
-
```

Comment:
Entire section *poof*?

Reply:
This was moved to line 159.

Reply:
Moved to bog_abilities_race.lst, where it makes more sense to have them.